tests/integration_tests/sql_lab/api_tests.py

```diff
@@ -51,7 +56,11 @@ class TestSqlLabApi(SupersetTestCase):
         clear=True,
     )
     def test_get_from_empty_bootsrap_data(self):
-        self.login(username="gamma_sqllab_no_data")
+        if utils.backend() == "postgres":
```

Comment:
@michael-s-molina I've spent/wasted hours trying to get a few failing PostgreSQL tests to pass. The tests succeed when testing locally and/or remotely when I run either i) the specific test, or ii) the specific file, however they fail when run as part of the entire test suite.

My hypothesis is that there are a slew of non-idempotent tests and this change merely (re)highlighted how fragile aspects of our tests are.

Reply:
Yeah, it's kind of frustrating that we have many non-idempotent tests. Given that we'll need a specific effort for fixing this problem globally, I'm ok with skipping the test. We do the same for many sqlite tests.

Reply:
Thanks @michael-s-molina. Also it's worth mentioning it's not like the test is being skipped globally, i.e., we have coverage on the said logic for MySQL and SQLite.